packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -139,6 +141,7 @@ export class SerializedStateManager {
 					supportGetSnapshotApi,
 				);
 				this.newSnapshotFetched?.();
+				this.sessionExpiryTimerStarted = Date.now();
```

Comment:
i think this needs to happen in updateSnapshotAndProcessedOpsMaybe and only happen if the snapshot is updated